Kajabi/sage-archived
# Stylelint repo cleanup

## Description
- [x] Revise build process to include stylelint task
- [x] Clean up errors in files

Note that any linting _errors_ will exit the session, but _warnings_ will just be listed.
We're using `sass-rails` to compile SCSS, whereas stylelint is run via `npm`, so saving/editing files during the session will _not_ update the linting in the console. For that workflow, we'll still be dependent on your code editor of choice to flag syntax and formatting issues.

## Test notes
### Steps for testing
1. Run `npm start` in your CLI
2. Note the linting warnings that are listed per file
3. Open a file in your code editor from the linting list above and verify that the errors/warnings match

## Related issues
- Resolves #75
